src/com/musala/xml/parser/Main.java

```diff
@@ -0,0 +1,181 @@
+package com.musala.xml.parser;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+import com.musala.xml.model.School;
+import com.musala.xml.model.SchoolClass;
+import com.musala.xml.model.Student;
+import com.musala.xml.model.Teacher;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException, JAXBException {
+        String pathname = "C:\\Users\\stefan.simeonov\\Desktop\\ThePath.xml";
+        Path pathname2 = Paths.get("C:\\Users\\stefan.simeonov\\Desktop\\ThePath.xml");
+        // File file = new File(pathname);
+        /*
+         * Scanner in = null;
+         * try {
+         * in = new Scanner(file);
+         * 
+         * while (in.hasNextLine()) {
+         * System.out.println(in.nextLine());
+         * }
+         * } catch (FileNotFoundException e) {
+         * System.out.println(e.getMessage());
+         * } finally {
+         * if (in != null)
+         * in.close();
+         * }
+         */
+
+        /*
+         * FileInputStream input = null;
+         * try {
+         * input = new FileInputStream(file);
+         * int content;
+         * while((content=input.read())!=-1){
+         * System.out.print((char)content);
+         * }
+         * } catch ( IOException ex) {
+         * System.out.println(ex.getMessage());
+         * }finally{
+         * if(input!=null) input.close();
+         * }
+         */
+        // FileReader inpt=new FileReader(file);
+        // char[] mass=new char[1000];
+        // inpt.read(mass);
+        // for(char symbol:mass){
+        // System.out.print(symbol);
+        // }
+        /*
+         * int content;
+         * while((content=inpt.read())!=-1){
+         * System.out.print((char)content);
+         * }
+         */
+        // byte[] arr = Files.readAllBytes(pathname2);
+        // System.out.println(new String(arr, "UTF-8"));
+        String pathname3 = "ThePath.xml";
+        getFile(Constants.filepath);
+
+        ClassLoader classload = Main.class.getClassLoader();
+        File configFile = new File(classload.getResource(pathname3).getFile());
+        JAXBContext jaxb = JAXBContext.newInstance(School.class);
+        Unmarshaller jaxbUnmarsheller = jaxb.createUnmarshaller();
+        School mySchool = (School) jaxbUnmarsheller.unmarshal(configFile);
+        System.out.println(mySchool.getStudents().getStudent().get(0).getAge());
+        System.out.println(mySchool.getStudents().getStudent().get(1).getAge());
+        System.out.println(mySchool.getSchoolClasses().get(0).getTeacherReference());
+        System.out.println(mySchool.getTeachers().getTeacher().get(0).getAge());
+        Print(mySchool);
+    }
+
+private static void Print(School myschool){
```

Comment:
The whole method is too long and does too many things. Please consider moving-out some of its logic in separate methods and calling them. An idea is to do the following: 
```java
 private static void Print(School myschool) {
        int numberOfClasses = myschool.getSchoolClasses().size();
        System.out.printf("The school name is: %s!%n There %s %d school classes.%n", myschool.getName(),
                numberOfClasses > 1 ? "are" : "is", numberOfClasses);
        
        printClassesData(myschool);

    }
    
    private static void printClassesData(School myschool) {
        //here implement the for (int i = 0; i < numberOfClasses; i++) {}...
        //inside the loop declare List<String> studentReferences =   myschool.getSchoolClasses().get(i).getStudentsReference();
        //again, inside the loop logic here, call printStudentReferenceData(studentReferences)
    }
    
    private static void printStudentReferenceData(List<String> studentReferences) {
        //implement the logic with for (int j = 0; j < studentReferences.size(); j++) {...
    }
```